### PR TITLE
demo application spring-jcache

### PR DESCRIPTION
**Pull request by mustafaiman**
Demo application which configures Spring to use JCache with Hazelcast backend.

### CODE REVIEW

File: hazelcast-integration/spring-jcache/pom.xml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>spring-jcache</artifactId>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-core</artifactId>
+            <version>4.1.0.RELEASE</version>
```
Comment (mesutcelik) [inline hazelcast-integration/spring-jcache/pom.xml]: can we use the latest stable 4.1.8 or 4.2.3?

File: hazelcast-integration/spring-jcache/src/main/java/com/hazelcast/spring/jcache/SpringJCache.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
+package com.hazelcast.spring.jcache;
+
+import com.hazelcast.client.HazelcastClient;
+import com.hazelcast.core.Hazelcast;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.support.GenericXmlApplicationContext;
+
+public class SpringJCache {
```
Comment (mesutcelik) [inline hazelcast-integration/spring-jcache/src/main/java/com/hazelcast/spring/jcache/SpringJCache.java]: Do we need any extra annotation to enable JCache on Spring?

**Comment by mesutcelik** [comment]
two minor comments so :+1:

File: hazelcast-integration/spring-jcache/src/main/java/com/hazelcast/spring/jcache/SpringJCache.java (hunk 2 above)
Comment (mustafaiman) [inline hazelcast-integration/spring-jcache/src/main/java/com/hazelcast/spring/jcache/SpringJCache.java]: `<cache:annotation-driven />` tag in `applicationContext.xml` does this.

File: hazelcast-integration/spring-jcache/pom.xml (hunk 1 above)
Comment (mustafaiman) [inline hazelcast-integration/spring-jcache/pom.xml]: sure I'll update the PR.

**Comment by mustafaiman** [comment]
I updated the PR @mesutcelik

**Pull request merged**